### PR TITLE
Add patched prompt directive that works on Safari and Chrome

### PR DESCRIPTION
**Pull request by jongiddy**
Upstream patch submitted as https://github.com/sbrunner/sphinx-prompt/pull/3

This temporarily uses the code from that patch in the Flocker code.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -34,7 +34,11 @@
     'sphinx.ext.ifconfig',
     'flocker.provision._sphinx',
     'flocker.docs.version_extensions',
-    'sphinx-prompt',
+    # Replace sphinx-prompt with patched version.  Patched version from
+    # https://github.com/sbrunner/sphinx-prompt/pull/3
+    # See FLOC-2102
+    # 'sphinx-prompt',
```
Comment (adamtheturtle) [inline docs/conf.py]: Get rid of commented out code, the issue is enough.

File: flocker/docs/prompt_patched.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# coding: utf-8
+
+
+from docutils import nodes
+from docutils.parsers import rst
+
+from pygments import highlight
```
Comment (adamtheturtle) [inline flocker/docs/prompt_patched.py]: I think that if we import something directly it should have a version-pinned dependency in setup.py

File: flocker/docs/prompt_patched.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# coding: utf-8
+
```
Comment (adamtheturtle) [inline flocker/docs/prompt_patched.py]: Docstring with comment and link to issue like in conf.py

**Comment by adamtheturtle** [comment]
Thanks @jongiddy.

Tested by viewing http://build.clusterhq.com/results/docs/sphinx-prompt-FLOC-1548/build-7547/using/installing/index.html on OS X 10.10.3 in Safari 8.0.6, Firefox 38.01 and Chrome 43. Unfortunately it showed up some bad prompts! I'll deal with that separately.

Hopefully the patch will be accepted soon. Address the minor comments and merge when you and the docs builder are happy.

**Comment by adamtheturtle** [comment]
@jongiddy As this has been merged into master upstream maybe we should just wait for a release? Depends on the timeframe.

**Comment by jongiddy** [comment]
Sounds like a few days at least. It's so easy to undo, I'll merge once tests have passed.

**Pull request merged**